src/theme/MDXComponents.ts

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import MDXComponents from "@theme-original/MDXComponents";
+
+import ContactLink from "@site/src/components/ContactLink";
+import SinceVersion from "@site/src/components/SinceVersion";
+import ChangelogFooter from "@site/src/components/ChangelogFooter";
+import Figure from "@site/src/components/Figure";
+import BrowserWindow from "@site/src/components/BrowserWindow";
+import Calendly from "@site/src/components/Calendly";
+import Description from "@site/src/components/Description";
+import BackportList from "@site/src/components/BackportList";
+
+export default {
+  ...MDXComponents,
+  ContactLink,
+  SinceVersion,
+  ChangelogFooter,
+  Figure,
+  BrowserWindow,
+  Calendly,
+  Description,
+  BackportList,
+};
```

Comment:
Doing builds locally I found a few instances where `SinceVersion` was used without importing it in the MDX file, this ensures the components are in the global MDX provider context, that way we no longer need to import them when using it.